doc/grass_database.html

```diff
@@ -222,18 +222,18 @@ <h3>Starting GRASS GIS using command line</h3>
 in a given Mapset with only command line interface:
 
 <div class="code"><pre>
-grass79 --text ~/grassdata/mylocation/mymapset
+grass80 --text ~/grassdata/mylocation/mymapset
```

Comment:
that's starting a text session in *nix. should we add an example how to do it in windows?

Reply:
Yes, please donate an example (here is fine)

Reply:
grass80 --text D:\grassdata\nc_spm_08_grass7\user1

Reply:
done in d83447807